Assignment/area_triangle.py

```diff
@@ -0,0 +1,28 @@
+class AreaTriangle:
+
+    def inputdetails(self):
```

Comment:
follow conventions of separating words input_details
better thing to do is 
the methods should be named as verbs , so the name enter_triangle_details is much better and self explanatory

Reply:
Noted and made the changes

Reply:
should have made the change to this file and not create a new file